smriprep/utils/misc.py

```diff
@@ -21,3 +21,50 @@ def apply_lut(in_dseg, lut, newpath=None):
                    segm.affine, hdr).to_filename(out_file)
 
     return out_file
+
+
+def fs_isRunning(subjects_dir, subject_id, mtime_tol=86400):
+    """
+    Checks FreeSurfer subjects dir for presence of recon-all blocking ``IsRunning`` files,
+    and optionally removes any based on the modification time.
+
+    Parameters
+    ----------
+    subjects_dir : os.PathLike or None
+        Existing FreeSurfer subjects directory
+    subject_id : str
+        Subject label
+    mtime_tol : int
+        Tolerance time (in seconds) between current time and last modification of ``recon-all.log``
+
+    Returns
+    -------
+    subjects_dir : os.PathLike or None
+
+    """
+    import logging
+    from pathlib import Path
+    import time
+
+    if subjects_dir is None:
+        return subjects_dir
+    subj_dir = Path(subjects_dir) / subject_id
+    if not subj_dir.exists():
+        return subjects_dir
+
+    isrunning = tuple(subj_dir.glob("**/IsRunning*"))
+    if not isrunning:
+        return subjects_dir
+    reconlog = subj_dir / "scripts" / "recon-all.log"
+    if not reconlog.is_file() or (time.time() - reconlog.stat().st_mtime) < mtime_tol:
```

Comment:
```suggestion
    if reconlog.is_file() and (time.time() - reconlog.stat().st_mtime) < mtime_tol:
```
Can we assume that if `reconlog` does not exist then the freesurfer folder is stale and the IsRunning files can safely be removed? (cc @effigies)

Reply:
i've changed the behavior in 76baa26